src/main/kotlin/edu/carroll/polyominoes/jpa/model/Account.kt

```diff
@@ -0,0 +1,57 @@
+package edu.carroll.polyominoes.jpa.model
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.OneToOne
+import jakarta.persistence.Table
+
+@Entity
+@Table(name = "account")
+class Account() {
+    companion object {
+        private val serialVersionUID: Long = 1L
+        private val EOL: String = System.lineSeparator()
+        private val TAB: String = "\t"
+    }
+
+    constructor(username: String, email: String, hashPassword: String) : this() {
+        this.username = username
+        this.email = email
+        this.hashPassword = hashPassword
+    }
+
+    @Id
+    @GeneratedValue
+    var id: Long? = null
+
+    @Column(name = "username", nullable = false, unique = true)
+    var username = ""
```

Comment:
I am NOT a fan of having to have default values (which is apparently a Kotlin behavior.

See
- https://stackoverflow.com/questions/32038177/kotlin-with-jpa-default-constructor-hell
- https://www.jpa-buddy.com/blog/best-practices-and-common-pitfalls/